master/rock.py

```diff
@@ -174,6 +177,11 @@ def CleanBuildCache(factory):
         command=["find", util.Interpolate(f"{CACHE_BUILD_BASE_DIR}/%(prop:target_buildername)s")]))
 
 def UpdateImportCache(factory, gem_compile=["ffi"]):
+    factory.addStep(steps.ShellCommand(
+        name="Fix permissions on /var/cache/autoproj",
+        command=["sudo", "chown", "buildbot", "-R", "/var/cache/autoproj"],
+        haltOnFailure=True
+    ))
```

Comment:
Why do you need this on every build? This may take a while as cache grows...

Reply:
I stopped trying to be smart about this ...  It comes back seemingly randomly, and since it's for the import cache update which runs rarely the impact is negligible in practice.